Mathlib/Tactic/Rename.lean

```diff
@@ -5,6 +5,11 @@ Authors: Gabriel Ebner
 -/
 import Lean
 
+/-!
+# The `rename'` tactic
+The `rename'` tactic renames one or several hypotheses. This file contains its definition.
```

Comment:
```suggestion
The `rename'` tactic renames one or several hypotheses.
```

Reply:
I had wondered about this, thanks for the clarification!